code/object/objectsnd.cpp

```diff
@@ -483,6 +483,9 @@ void obj_snd_do_frame()
 		}
 
 		gs = gamesnd_get_game_sound(osp->id);
```

Comment:
Would a `nullptr` check with `UNREACHABLE()` be worthwhile?

It's weird that `gs` is retrieved here but not accessed for another ~100 lines. 🤷‍♂️

Reply:
I don't think a `nullptr` check is warranted here, given that the underlying operation should never return null.

I like that `gs` is retrieved and checked here; if it's invalid, a lot of work is skipped.